README.md

```diff
@@ -104,8 +104,9 @@ noise_reduction: default
 - `mic` is a special option to control your microphone's input level _(uses the default recording device)_
 - **_New:_** `deej.unmapped` is a special option to control all apps that aren't bound to any slider ("everything else") (_experimental_)
 - **_New:_** On Windows, `deej.current` is a special option to control whichever app is currently in focus (_experimental_)
-- On Windows, you can specify a device's full name, i.e. `Speakers (Realtek High Definition Audio)`, to bind that device's level to a slider. This doesn't conflict with the default `master` and `mic` options, and works for both input and output devices.
-  - Be sure to use the full device name, as seen in the menu that comes up when left-clicking the speaker icon in the tray menu
+- You can specify a device's full name, i.e. `Speakers (Realtek High Definition Audio)`, to bind that device's level to a slider. This doesn't conflict with the default `master` and `mic` options, and works for both input and output devices.
+  - Windows: Use the full device name, as seen in the menu that comes up when left-clicking the speaker icon in the tray menu
+  - Linux (_expirimental_): Use the full device name, as seen in the `Output Devices` tab in `pavucontrol` or the description when running `pactl list sinks`
```

Comment:
- Typo (same as above)
- I would prefer to distinguish the difference by writing something like:
```
Use the device's description, as seen in PulseAudio's output devices tab (or by running `pactl list sinks`)
```